arcaflow_plugin_horreum_client/horreum_client_plugin.py

```diff
@@ -34,6 +34,11 @@ def horreum_client(
         auth_return.raise_for_status()
         token = json.loads(auth_return.text)["access_token"]
 
+    except requests.exceptions.SSLError as e:
+        return "error", ErrorOutput(
+            f"Unable to establish SSL connection with the Keycloak server: {e}"
+        )
+
     except (requests.ConnectionError, requests.HTTPError, requests.Timeout):
```

Comment:
I think that only the `HTTPError` can result from `raise_for_status` ... in the other cases there won't be an `auth_return` to dereference.

Reply:
Updated.